README.md

```diff
@@ -64,7 +64,7 @@ brew install caf --HEAD
 ### Conan
 
 A [Conan](https://conan.io/) recipe for CAF along with pre-built libraries
-for most platforms are available at [bincrafters/caf](https://bintray.com/bincrafters/public-conan/caf%3Abincrafters).
+for most platforms are available at [Conan Center](https://conan.io/center/caf/0.17.3/?user=bincrafters&channel=stable).
```

Comment:
I'm not familiar with Conan Center, but is there a way to link to the latest version instead of picking a release version? We'd basically have to update the `README.md` whenever releasing a new CAF version.

The link https://conan.io/center/caf/stable/?user=bincrafters&channel=stable works for me, but I'm not sure if this is how you're supposed to link to a project. In fact, I can replace `0.17.3` in the URL with any string and still get to the right page.

Reply:
Yes, you're absolutely right.  Fixed.